go-controller/pkg/ovn/external_ids_syncer/logical_router_policy/logical_router_policy_sync.go

```diff
@@ -101,7 +101,8 @@ func (syncer *LRPSyncer) syncEgressIPReRoutes() error {
 			podInfo, err := cache.getPod(podIP)
 			if err != nil {
 				klog.Infof("Failed to find Logical Switch Port cache entry for pod IP %s: %v", podIP.String(), err)
-				continue
+				// pod not found, add dummy metadata that will be cleaned up by EIP controller sync.
+				podInfo = podNetInfo{namespace: "UNKNOWN", name: "UNKNOWN"}
```

Comment:
so you set these values as UNKNOWN, then that generates the corresponding external ids, then you update the policy with this? I'm not following how this triggers the syncer to remove it.

Reply:
when eip syncer starts then it wont find a namespace + pod of that name (its also not possible)
I didnt want to do removal within this func or pkg and let it up to the eip controller.

Reply:
eip controller will only react to ovn constructs it owns - thats why i added this dummy external IDs with the owner being the eip controller.